t/31-api_descriptions.t

```diff
@@ -18,8 +18,11 @@ use Test::Most;
 use Test::Mojo;
 use Test::Warnings ':report_warnings';
 
+use OpenQA::Test::Case;
+
+OpenQA::Test::Case->new->init_data(skip_fixtures => 1);
+my $t = Test::Mojo->new('OpenQA::WebAPI');
 use_ok('OpenQA::WebAPI::Description', qw(get_pod_from_controllers set_api_desc));
-$ENV{OPENQA_CODEBASE} = ".";
-get_pod_from_controllers();
+get_pod_from_controllers($t->app);
```

Comment:
It is not nice that we initialize the database here (even without fixtures) just for passing the app. Maybe it works to just pass a generic app instantiated via `Mojolicious->new` here?